File: connect.py
#C:\Users\Gantasoft\AppData\Local\Programs\Python\Python312
print("Cpntent-Type:text/html")
print()
import cgi

print("<h1>Welcome to the Python</h1>")
print("<hr/>")
print("<h1>Using input tag</h1>")
print("<body bgcolor='red'>")

form = cgi.FieldStorage()

first_name=form.getvalue("fname")
last_name=form.getvalue("lname")
mobile_number=form.getvalue("number")
email=form.getvalue("email")
gender=form.getvalue("gender")
user_password = form.getvalue("password")

import mysql.connector

con=mysql.connector.connect(user='root', password='', host='localhost', database='tirupathiraosesapu')
cur=con.cursor()

cur.execute("insert into login_information values (%s,%s,%s,%s,%s,%s)",(first_name, last_name, gender, email, mobile_number, user_password))
cur=con.commit()

cur.close()
con.close()
print("<h3>Record Inserted succefully</h3>")
print("<a href=''>Click here to go back</a>")

File: connect.php
<?php 
    $serverName="localhost";
    $userName="root";
    $password = "";
    $databaseName="tirupathiraosesapu";

    $conn = mysqli_connect($serverName, $userName, $password, $databaseName);

    // Database connection
    if (!$conn) {
            die("Connection Failed :".mysqli_connect_error());
    }

    if (isset($_POST['save']))
    {
        $first_name = $_POST['fname'];
        $last_name = $_POST['lname'];
        $email = $_POST['email'];
        $number = $_POST['number'];
        $gender = $_POST['gender'];
        $pasword = $_POST['password'];

        $sql_query = "INSERT INTO login_information (first_name, last_name, email, gender, mobile_number, user_password) values ($first_name, $last_name, $email, $gender, $number, $pasword)";

        if (mysqli_query($conn, $sql_query))
        {
            echo "New Details Entry Inserted Successfully !";
        }
        else
        {
            echo "Error :".$sql."".mysqli_error($conn);
        }
        mysqli_close($conn);
    }
?>